fvtr/fvtr.sh

```diff
@@ -35,16 +35,23 @@ CONFIGFILE=""
 
 function usage () {
 	echo "Usage: fvtr [-fl] <configfile> [test1 test2 ...]"
+	echo "       fvtr report"
+	echo ""
 	echo "       -f Force the test(s) to be run"
 	echo "       -l Output is sent to the console"
-
+	
 	echo "       '<configfile>' path to the config file used to build AT"
 	echo "       tests -  a list of tests to run (optional).  It is going"
 	echo "                to run all tests by default."
 	echo "       Example: fvtr -f ~/advance-toolchain/config.at5.0.sles10 "
-	echo ""
-	echo "   or: fvtr report"
-	echo "       the report subcommand outputs the logs of failed tests"
+	echo "       report - prints content of log files that failed. This"
+	echo "                command does not run tests again. Instead, it"
+	echo "                only takes the logs of the last test that was"
+	echo "                ran with fvtr.sh in order to print relevant"
+	echo "                information. It's also important to note that"
+	echo "                this implies that a test must be necessarily"
+	echo "                be ran with the -l option in order to enable later"
```

Comment:
This line have more than 79 characters (see https://www.gnu.org/prep/standards/standards.html#Formatting).